src/app/not-found.tsx

```diff
@@ -3,10 +3,16 @@
 import Image from 'next/image';
 import { useRouter } from 'next/navigation';
 
+import NextButton from '@/components/common/NextButton';
+
 const NotFoundPage = () => {
   const router = useRouter();
+
+  const goHome = () => {
```

Comment:
저희 함수에 대해서 컨벤션 정할까요? 

저는 회사에서 보통 `handle` 이라는 `prefix`를 사용하긴하는데 투머치라고 생각하시면 넘어가겠습니다.

```suggestion
  const handleGohome = () => {}
```

Reply:
handle로 가죠~

Reply:
다음 PR에 한번에 수정할께용